src/core/src/pass/serialize.cpp

```diff
@@ -696,74 +696,6 @@ bool has_dynamic_output(const std::shared_ptr<Node>& n) {
     return false;
 }
 
-bool resolve_dynamic_shapes(const ngraph::Function& f) {
```

Comment:
Please add test for Serialization with dynamic dims.

Reply:
Where is the place to add it?

Reply:
openvino/inference-engine/tests/functional/inference_engine/ir_serialization/
Look at partial_shape_serialization.cpp, partial_shape_deserialization.cpp as examples.

Reply:
As far as I can understand this functionality covered by tests from here: `src/tests/functional/inference_engine/ir_serialization/partial_shape_serialization.cpp`. And looks like we already test dynamic dims and these tests passed without changes.